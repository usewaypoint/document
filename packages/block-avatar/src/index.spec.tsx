import React from 'react';

import { render } from '@testing-library/react';

import { Avatar } from '.';

describe('block-avatar', () => {
  describe('Avatar', () => {
    it('renders with default values', () => {
      expect(render(<Avatar />).asFragment()).toMatchSnapshot();
    });
  });
});
