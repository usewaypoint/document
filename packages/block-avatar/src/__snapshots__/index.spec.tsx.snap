// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block-avatar Avatar renders with default values 1`] = `
<DocumentFragment>
  <div>
    <img
      alt=""
      height="64"
      src=""
      style="outline: none; text-decoration: none; object-fit: cover; height: 64px; width: 64px; max-width: 100%; display: inline-block; vertical-align: middle; text-align: center;"
      width="64"
    />
  </div>
</DocumentFragment>
`;
